### PR TITLE
Category filter

### PR DESCRIPTION
**Pull request by tiagoit**
Adds a filter by category component.
When a category is selected the list of books is updated to only display the books that belong to the chosen category.

### CODE REVIEW

File: src/containers/CategoryFilter.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import categories from '../constants/categories';
+
+const CategoryFilter = (props) => {
```
Comment (bellom) [inline src/containers/CategoryFilter.js]: Our `CategoryFilter.js` should be placed in src/components directory.
image
`you'll add a new presentational component (CategoryFilter) which will just be a simple select box.`

For any file not connecting to the redux store should be in presentational component folder. Components in presentational Component folder do not have direct access to Redux or other data stores.
https://www.reactnative.guide/9-redux/9.2-presentational-vs-containers.html

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi Team :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1:, just two changes required............ :slightly_smiling_face: 
**Note:** This `CRR` should be mark as `invalid` in the first place, because it doesn't look professional
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/

**Kindly make sure you go through the above 👆link before working on the following;** 

1. You need to add linter's check on your `github` page
add linter's check and fix all errors, checkout this [linters-config#troubleshooting](https://github.com/microverseinc/linters-config#troubleshooting) for help. Example below;
image
If you are not having this check on your `Github` page, kindly visit :point_right:[ linter set up](https://github.com/microverseinc/linters-config/blob/master/ruby/.stickler.yml)

2. Kindly checkout the comment below

3. update your readme file


**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Comment by tiagoit** [comment]
@bellom
image

Stickler was working on the previous branches, there is no change on `.stickler`, I've disabled and re-enabled it. Seem like a bug from the tool. I've ran eslint on my local env and everyting looks ok.

**Comment by bellom** [comment]
@tiagoit  can we jump on call and look into it together..... ?

**Review by jofer86 — APPROVED** [review]
## 👋 👋 Hello Team!!! Everything is looking good!!! it is time to move on!! 👍 👍  Approved!

**Comment by tiagoit** [comment]
@bellom sure, I'm messaging you on Slack.

**Comment by bellom** [comment]
@tiagoit replied !

**Pull request merged**